showcase/src/app/demo/logout/logout.component.html

```diff
@@ -0,0 +1,7 @@
+<example-viewer [extensions]="['HTML']" filesPath="logout/logout-default" exampleTitle="Logout button - Default icon">
+	<stark-app-logout></stark-app-logout>
+</example-viewer>
+
+<example-viewer [extensions]="['HTML']" filesPath="logout/logout-logout" exampleTitle="Logout button - Custom icon">
```

Comment:
Can you add the translations in FR, EN and NL for the "exampleTitle"? They should be in the files under `showcase/src/assets/translations`